imperial_coldfront_plugin/migrations/0014_auto_20250508_2158.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.11 on 2025-05-08 21:58
+
+from django.db import migrations
+
+def schedule_update_quota_usages_task(apps, schema_editor):
+    Schedule = apps.get_model("django_q", "Schedule")
+    Schedule.objects.update_or_create(
+        name="Update quota usages",
+        func="imperial_coldfront_plugin.gpfs_client._update_quota_usages_task",
+        schedule_type="H",
+    )
```

Comment:
Are these things typically meant to be added manually to migrations? Seems cumbersome.

Reply:
I think it's the best way to handle them as they are database entries. The only alternative would be to just stick it somewhere where it would get created during the app initialisation process but that means you don't have clean options to remove it later.